src/_http.ts

```diff
@@ -143,12 +157,36 @@ interface ChalkErrorData {
   resolver?: string;
 }
 
+export interface ChalkOnlineQueryRawData {
```

Comment:
It might make sense to move some of these types to `_types.ts`? I don't have a strong opinion.